releasability_check/src/main.py

```diff
@@ -33,16 +33,17 @@ def releasability_check(request: Request):
     if not paths or len(paths) != 4:
         return make_response("Bad Request", 400)
 
-    organization = paths[1]
-    project = paths[2]
-    version = paths[3]
+    _, organization, project, version = paths
     if organization != "SonarSource":
         return make_response("Unauthorized organization", 403)
 
+    # Read branch from optional ?branch=xxx/yyy/zzz parameter
+    branch = request.args.get('branch', 'master')
```

Comment:
why do you hardcode master here?

Reply:
oh it's a default value, my bad

Reply:
To be completely accurate, we would have to interrogate git to know the "default" branch - but for our projects we know it's master :smile: